### PR TITLE
Bac 31

### PR DESCRIPTION
**Pull request by aosyborg**

### CODE REVIEW

File: src/app/landing/landing.html [hunk 1]
```diff
@@ -45,38 +45,29 @@
...
-                    <button type="button" class="btn btn-category">Technology</button>
-                    <button type="button" class="btn btn-category">Theater</button>
-                </div>
-
-                <div class="notification-slider">
-                    <h4>How often do you want to be notified?</h4>
-                    <input id="notificationSlider" data-slider-id='notificationSlider1' type="text" data-slider-min="0" data-slider-max="20" data-slider-step="1" data-slider-value="14">
+                <div class="category-listing">
+                    <button type="button" class="btn btn-category" toggle-class='active' ng-click="vm.selectCategory('art')">Art</button>
+                    <button type="button" class="btn btn-category" toggle-class='active' ng-click="vm.selectCategory('comics')">Comis</button>
```
Comment (printscreen) [inline src/app/landing/landing.html]: comis
comis in Technology

language
A COMpilation and Interpretation System.
A Fortran interpreter use by the PAW system.
(1994-11-30)
The Free On-line Dictionary of Computing, © Denis Howe 2010 http://foldoc.org
Cite This Source

File: bower.json [hunk 2]
```diff
@@ -21,7 +21,8 @@
     "font-awesome": "~4.3.0",
     "angular-bootstrap": "~0.13.0",
     "momentjs": "~2.10.2",
-    "angular-moment": "~0.10.1"
+    "angular-moment": "~0.10.1",
+    "jasmine-jquery": "~2.1.0"
```
Comment (EnnEssEss) [inline bower.json]: Why did you add this?

File: src/app/landing/landing.html [hunk 3]
```diff
@@ -45,38 +45,29 @@
...
-                    <button type="button" class="btn btn-category">Technology</button>
-                    <button type="button" class="btn btn-category">Theater</button>
-                </div>
-
-                <div class="notification-slider">
-                    <h4>How often do you want to be notified?</h4>
-                    <input id="notificationSlider" data-slider-id='notificationSlider1' type="text" data-slider-min="0" data-slider-max="20" data-slider-step="1" data-slider-value="14">
+                <div class="category-listing">
+                    <button type="button" class="btn btn-category" toggle-class='active' ng-click="vm.selectCategory('art')">Art</button>
+                    <button type="button" class="btn btn-category" toggle-class='active' ng-click="vm.selectCategory('comics')">Comics</button>
```
Comment (EnnEssEss) [inline src/app/landing/landing.html]: I don't like this toggle-class directive thing that you have going on.  You should be tying the active class on an element back to the data that's being tracked in the controller.

```ng-class="{active: vm.isActiveCategory('comics')}"```
